### PR TITLE
Removed puzzle

### PR DESCRIPTION
**Pull request by cyberone**

### CODE REVIEW

**Comment by dmarkov** [comment]
I'll find a reviewer for your pull requests shortly, thanks for contribution!

**Comment by dmarkov** [comment]
@carlosmiranda could you please review this PR

File: src/main/java/com/jcabi/github/User.java [hunk 1]
```diff
@@ -91,13 +91,6 @@
 
     /**
      * Smart user with extra features.
-     * @todo #1:30min Implement methods to retrieve all values provided
-     *  by Github for a single user, see:
-     *  http://developer.github.com/v3/users/#get-a-single-user
-     *  At the moment we implement just a few, but every data
-     *  items should have its own method. Of course, every new item should
-     *  be tested by a new unit test method.
-     * @see <a href="http://developer.github.com/v3/users/#get-a-single-user">Get a Single User</a>
```
Comment (carlosmiranda) [inline src/main/java/com/jcabi/github/User.java]: Why was this line removed? It's not part of the puzzle.

**Comment by carlosmiranda** [comment]
@cyberone Just one comment above.

**Comment by cyberone** [comment]
@carlosmiranda I've returned the line. Please re-check.

**Comment by carlosmiranda** [comment]
@cyberone Looks good now, thank you

**Comment by carlosmiranda** [comment]
@rultor Merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by dmarkov** [comment]
@carlosmiranda Much obliged! I've added **16 mins** to your account in payment "50517774". +16 added to your rating, at the moment it is: [+1438](http://www.netbout.com/b/35023?open=rating)

**Comment by dmarkov** [comment]
@rultor please deploy
